### ISSUE TITLE
Broken Contribute link in README

### ISSUE THREAD

**Issue by Calinou** [vaxerski/Hyprland#212]
The Contribute menu link in the README isn't working: https://github.com/vaxerski/Hyprland/blob/main/docs/Contribute.md

**Comment by siemato** [comment]
Yeah, that thing has now been worked into [Issue Guidelines](https://github.com/vaxerski/Hyprland/blob/a4e21a25f4d7f60bdf35cb033da1b5a64429ad02/docs/ISSUE_GUIDELINES.md)

**Comment by vaxerski** [comment]
wasnt, worked into issue guidelines, but fixed. thanks!
